### ISSUE TITLE
 all DNS records got deleted when IPv4 not detected

### ISSUE THREAD

**Issue by dechunwang** [timothymiller/cloudflare-ddns#60]
**Describe the bug**
Hi there, I had facing a problem where DDNS accidentally (randomly ) deleted all my DNS records on my site. 
This happened  three times on three different machines  that installs  ddns scripts.
I set up the config as  in Doc:
```
{
  "cloudflare": [
    {
      "authentication": {
          "api_token": "xxx" 
      },
      "zone_id": "xxx",
      "subdomains": [
        "arm"
      ],
      "proxied": false
    }
  ],
  "a": true,
  "aaaa": false
}
```

**To Reproduce**
Crontab calling start-sync.sh  every 4 mins
**Expected behavior**
should only update subdomain's ip address

**Screenshots**
the log  output from the script:
![image](https://user-images.githubusercontent.com/16343127/122520280-fbc14400-d045-11eb-8ef8-c3569dd5d19f.png)
it clear shows when IPv4 not detected, the program deleted all sub-domain  dns record of my site.

here is the log in clouldflare:
![image](https://user-images.githubusercontent.com/16343127/122520451-31662d00-d046-11eb-98ce-b49ae9b22133.png)

**Desktop (please complete the following information):**
 - OS: [Ubuntu 20.04]

**Smartphone (please complete the following information):**
N/A

**Additional context**
N/A

**Comment by JCAguilera** [comment]
Same happened to me today. Have you found any workarounds or fixes?

**Comment by ImTung** [comment]
Same :(

**Comment by jesswalters-wk** [comment]
Just had the same thing happen to me 😢

**Comment by jesswalters-wk** [comment]
If it helps anyone else, I made a docker  image that pulls in #57 so that you can turn off that functionality

**Comment by anthonydiiorio** [comment]
Same. Was able to restore records from the audit log thankfully.

**Comment by kikkia** [comment]
I forked the repo and packaged a proposed pr to fix this. Until the pr is merged feel free to use this docker image `registry.gitlab.com/kikkia/cloudflare-ddns` its all open source if you want to verify what was changed.
https://gitlab.com/kikkia/cloudflare-ddns

**Comment by timothymiller** [comment]
This has been fixed in PR #57
